bin/setup

```diff
@@ -0,0 +1,66 @@
+f this is a vagrant VM, cd to ~/opencraft on login
+if [[ $USER == 'vagrant' ]]; then
+    mkdir -p ~/opencraft
+    cd ~/opencraft
+    grep -Fq 'cd ~/opencraft' ~/.bashrc || echo 'cd ~/opencraft' >> ~/.bashrc
+fi
+
+# Need python-3.6 repository
+echo "Setting up python3.6 repository"
+sudo add-apt-repository ppa:jonathonf/python-3.6
+sudo apt-get update
+
+sudo apt-get install -y python3.6 python3.6-dev
+
+# Install pip for python3
+echo "Installing pip for python3"
+sudo apt-get install -y python3-pip
+sudo pip3 install --upgrade pip
+sudo pip3 install --upgrade virtualenv
+
+# Install all dependencies.
+cd ~/opencraft
+echo "Installing dependencies."
+sudo -E apt-get -y autoremove
+for i in `tr -d '\r' < debian_packages.lst`; do sudo apt-get install -y $i; done
+
+# Activate VirtualEnv
+cd ~/
+mkdir -p ~/.virtualenvs
+virtualenv -p python3.6 ~/.virtualenvs/opencraft
+source ~/.virtualenvs/opencraft/bin/activate
+
+# Need wkhtmlox
```

Comment:
`sed -i 's/wkhtmlox/wkhtmltopdf/' bin/setup`. (WebKit HTML to PDF).

Reply:
You seem confused. If you take a look at the download link at https://wkhtmltopdf.org/downloads.html you'll see that's the download for wkhtmltopdf.

Reply:
With the upgrade in ubuntu I can remove all this code now.